lru_cache/lru_cache.py

```diff
@@ -79,3 +79,12 @@ def set(self, key, value):
                     self.cache[k].value = None
 
             self.count -= 1
+
+
+# QUESTION: Because a dictionary (cache look-up) is not a true hash table, the RTC is O(n)
+# So if we never delete entries from the dictionary (cache look-up) this cache will grow
+# and grow and grow.. defeating the whole purpose of fast access to recently used data
+
```

Comment:
You can delete the entries every time when setting new key-pair values. I forgot to do this in mine and but my tests pass. I was like wait for a second, this dictionary is gonna be so big, and isn't working as a true cache.